docs/docs/0.7.1/documentation/security.md

```diff
@@ -63,19 +55,39 @@ $ kuma-dp run \
   --dataplane-token-file=/tmp/kuma-dp-echo-1-token
 ```
 
-#### Accessing Admin Server from a different machine
+You can also pass Dataplane Token as `KUMA_DATAPLANE_RUNTIME_TOKEN` Environment Variable. 
+
+#### Dataplane Token boundary
+
+As you can see in the example above, you can generate a token by passing name, mesh, and tags. Control Plane will verify Dataplane against data available in the Token. This means you can generate a token by specifying:
+* only `mesh`. This way you can reuse the token for all dataplanes in a given mesh.
```

Comment:
Do we have a practical use-case where one would want `tag` only tokens? Or maybe that is not a good practice and we should not advise for it?

Reply:
I don't think placing dp token on only a tag is useful.